client/my-sites/subscribers/components/jetpack-empty-list-view/jetpack-empty-list-view.tsx

```diff
@@ -86,7 +86,7 @@ const JetpackEmptyListView = () => {
 			</h2>
 			<p className="jetpack-empty-list-view__description">
 				{ i18n.fixMe( {
-					text: translate( 'No subscribers yet? Turn your site visitors into subscribers.' ),
+					text: 'No subscribers yet? Turn your site visitors into subscribers.',
```

Comment:
```suggestion
					text: 'No subscribers yet? {{howToTurnVisitorsLink}}Turn your site visitors into subscribers.{{/howToTurnVisitorsLink}}',
```

I might be missing something, but that's the thing you are checking for translations, no? I can see this in GlotPress partly translated 🤔

Reply:
You're right! I thought I was supposed to remove components in `text` for some reason.

Reply:
Ohh yes - I can see that can be misleading. We can probably update the description, perhaps even the property name 🤔